tools/sup-network/Makefile

```diff
@@ -0,0 +1,50 @@
+ifneq (${HAB_SUP_TEST_NETWORK_LOG_SERVICE},)
+	log_service := ${HAB_SUP_TEST_NETWORK_LOG_SERVICE}
+else
+	log_service := rando
+endif
```

Comment:
This can be simplified to just
```make
log_service := rando
```
Any assignments made with `:=` are overridden by assignments on the command line. For example, given the `Makefile`:

```make
BAR := baz
foo:
	@echo ${BAR}

.PHONY: foo
```
```bash
➤ make foo
BAR is 'baz'

➤ make foo BAR=qux
BAR is 'qux'
```

Similarly elsewhere

Reply:
Thanks, that's great to know.  My incomplete knowledge of `Makefile`s is showing.

Reply:
Although I'm assuming in this case, I'd want to do something like

`HAB_SUP_TEST_NETWORK_LOG_SERVICE := rando`

and then just refer to `${HAB_SUP_TEST_NETWORK_LOG_SERVICE}` elsewhere in the file, rather than `${log_service}`

Reply:
I'm fascinated to note that your example:

`make foo BAR=qux`

works but

`env BAR=qux make foo`

doesn't.

Reply:
Yeah, this is for make variables, not environment variables. One could make it work with environment variables, but I think it's more painful.

Since `HAB_SUP_TEST_NETWORK_LOG_SERVICE` isn't needed elsewhere as an env var, I'd be inclined to use `log_service := rando` and then overrides on the CLI take the form
```sh
$ make logs log_service=whatever
```